ubiquity/dell-bootstrap.py

```diff
@@ -760,6 +761,16 @@ def prepare(self, unfiltered=False):
             self.log("rec_type %s, stage %d, device %s" % (rec_type, self.stage, self.device))
             if (rec_type == 'factory' and self.stage == 2) or rec_type == 'hdd':
                 self.fixup_factory_devices(rec_part)
+            if rec_type == 'hdd':
+                # copy old mok key so that user don't need to enroll it again.
+                rootfs = mount[0:-1] + '3'
+                self.log("old rootfs from %s" % rootfs)
+                misc.execute_root('mount', '-o', 'ro', rootfs, '/mnt')
+                if os.path.exists('/mnt/var/lib/shim-signed/mok/MOK.priv') and os.path.exists('/mnt/var/lib/shim-signed/mok/MOK.der'):
+                    misc.execute_root('cp', '-f', '/mnt/var/lib/shim-signed/mok/MOK.der','/tmp')
```

Comment:
Why not use shutil.copy?

Reply:
because MOK.priv need root privilege,  execute_root seems a convenient tool to achieve my target.

Reply:
Better style is to use:

```
with misc.raised_privileges():
```

So that native python libraries are usable.